### PR TITLE
authored releasing.md

### PR DESCRIPTION
**Pull request by thoemmi**
As requested by @dblock in #67 I have documented the steps to create a release.

### CODE REVIEW

**Comment by dblock** [comment]
👍

File: RELEASING.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+* Add the next release to [CHANGELOG.md](CHANGELOG.md).
+
+  ```
+  ### 2.1 (Next)
+
+  * Your contribution here.
+  ```
+
+* Increment the third version number in [lib/iex/version.rb](lib/iex/version.rb).
```
Comment (icnocop) [inline RELEASING.md]: This file doesn't exist in the resourcelib repo.

Comment (dblock) [reply RELEASING.md]: I did a sloppy job reviewing this :)

File: RELEASING.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# Releasing resourcelib
+
+There're no hard rules about when to release resourcelib. Release bug fixes
+frequently, features not so frequently and breaking API changes rarely.
+
+## Release
+
+* Verify that all issues assigned to the [milestone](https://github.com/resourcelib/resourcelib/milestones)
```
[the remark this answers is not in the document]
Comment (thoemmi) [reply RELEASING.md]: @hakito No need to close a milestone, it's just a collection of issues and PRs.

**Pull request merged**